### ISSUE TITLE
Best Buds Widget Code Update

### ISSUE THREAD

**Issue by LCLambert** [denverpost/cannabist#12]
Hi Joe,

Here is the latest widget code updates to correct a few issues Aleta and Rico shared. 

```
<a href="http://www.thecannabist.co/map/"><img src="http://bestbudsapp.com/sites/default/files/cannabist_map_new.jpg"></a>
<form style="margin-top:15px;font-family: Arial;font-size:13px;" type="post" action="http://www.thecannabist.co/map/">
    <input type="hidden" name="field_dispensary_type_value" value="All">
    <input type="text" name="field_geofield_distance[origin]" style="float: left; padding:5px;width:40%;margin-right:5px" placeholder="City, State or Zip" id="origin">
    <input type="text" name="title" style="float: left; padding:5px;width:40%;margin-right:5px" placeholder="Strain or Product Name" id="title">
    <button style="display: inline-block; border: 0px none; background: none repeat scroll 0px 0px rgb(92, 184, 92); color: rgb(255, 255, 255); font-size: 12px; padding: 6px 12px;" type="submit">Submit</button>
</form>
<p style="font-family: Arial;font-size:13px;color:#888;">Not listed on the map? <a href="http://bestbudsapp.com/dispensary/register">Click here</a> to register your store.</p>
```

**Comment by freejoe76** [comment]
Okay this new markup is live on http://www.thecannabist.co/

**Comment by LCLambert** [comment]
Thanks again Joe.

On Thursday, February 5, 2015, Joe Murphy <notifications@github.com> wrote:

> Closed #12 <https://github.com/denverpost/cannabist/issues/12>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/denverpost/cannabist/issues/12#event-231193695>.
>


-- 
Lance C. Lambert
Director of Digital Products & Platforms
Digital First Media | The Denver Post

LcLambert@DenverPost.com
o: 303.954.1711 | c: 805.746.1921

.:iPhone mail:.
